doc/kubernetes/modules/ROOT/pages/running/metrics/keycloak_cluster.adoc

```diff
@@ -0,0 +1,27 @@
+= {project_name} cluster deployment
+:description: This document contains details of the metrics to monitor your {project_name} deployment's performance.
+:jgrp_application: {project_name}
+:clustered_caches: true
+
+{description}
+
+The deployment described in this document is for single-site deployments.
+In this architecture, {project_name} nodes creates cluster between themselves, used by the {ispn} clustered caches.
```

Comment:
"In this architecture, {project_name} nodes leverage  {ispn} embedded clustered caches to form a cluster.